data/messages.json

```diff
@@ -114,11 +114,18 @@
   "COMMUNICATION_WORLD_MESSAGE": "{%s}[%d][World: %d] %s:{FFFFFF} %s",
 
   "DEATH_MATCH_AVAILABLE_ZONES" : "Available DM zones are: /deathmatch %s.",
+  "DEATH_MATCH_DAMAGE_ACCURACY": "You've done %d damage with a %d percent accuracy.",
```

Comment:
Are these messages intentionally white? Should they get some colour that distinguishes them for players, based on category of functionality? We probably want to use the minigame colour.